packages/manager/apps/web/client/app/domain/zone/record/domain-zone-record.controller.js

```diff
@@ -768,5 +769,13 @@ angular.module('App').controller(
           this.$scope.resetAction();
         });
     }
+
+    static getSPFIncludeName() {
+      return `mx.ovh.${
+        Environment.constructor.isCanadaRegion(Environment.getRegion())
+          ? 'ca'
+          : 'com'
+      }`;
+    }
```

Comment:
```js
return `mx.ovh.${Environment.getRegion() === 'ca' ? 'ca' : 'com'}`
```
just do that, no need to change Environement

Reply:
Ok changed